docs/nidigital/class.rst

```diff
@@ -2471,28 +2471,25 @@ send_software_edge_trigger
 
     .. py:currentmodule:: nidigital.Session
 
-    .. py:method:: send_software_edge_trigger(trigger, trigger_identifier)
-
-            TBD
-
-            
+    .. py:method:: send_software_edge_trigger()
 
+        Sends a command to trigger the signal generator. This VI can act as an
```

Comment:
this device is not a signal generator

instrument?

Reply:
Used "Digital Pattern Instrument" in the docstring so using that here.